drivers/wireless/ieee80211/bcm43xxx/bcmf_sdpcm.c

```diff
@@ -134,75 +138,133 @@ int bcmf_sdpcm_process_header(FAR struct bcmf_sdio_dev_s *sbus,
 
 int bcmf_sdpcm_readframe(FAR struct bcmf_dev_s *priv)
 {
+  struct bcmf_sdpcm_header tmp_hdr;
```

Comment:
Please move this line to below "struct bcmf_sdpcm_header *header;"

Reply:
Hi Alan,
I've moved it.